tests/test_auditing.py

```diff
@@ -385,3 +392,106 @@ def test_create_commit_message(migration_model: capellambse.MelodyModel):
         "- connector v1\n"
         f"- RM-Bridge v{__version__}\n"
     )
+
+
+@pytest.mark.integtest
+def test_create_commit_message_on_no_changes(
+    migration_model: capellambse.MelodyModel,
+):
+    context: list[tuple[list[auditing.Change], str, str]] = [
+        ([], "example", "category2")
+    ]
+    tool_metadata = {
+        "revision": "123",
+        "tool": "tool version 1",
+        "connector": "connector v1",
+    }
+
+    reporter = auditing.RMReporter(migration_model)
+    for changes, module_id, module_category in context:
+        reporter.store_changes(changes, module_id, module_category)
+    commit_message = reporter.create_commit_message(tool_metadata)
+
+    assert commit_message.startswith(
+        "No changes identified from rev.123\n"
+        "\n"
+        "There were no modifications, extensions or deletions from "
+        "the previous revision of RM content.\n"
+        "\n"
+        "This was done using:\n"
+        "- tool version 1\n"
+        "- connector v1\n"
+        f"- RM-Bridge v{__version__}\n"
+    )
+
+
+def test_create_commit_message_raises_ValueError_on_faulty_parent(
+    migration_model: capellambse.MelodyModel,
+):
+    req = migration_model.by_uuid(TEST_REQ_UUID)
+    req.parent.requirements.remove(req)
+    changes: list[auditing.Change] = [
+        auditing.Deletion(
+            module=TEST_MODULE_ID,
+            parent=TEST_FOLDER_UUID,
+            attribute="requirements",
+            element=f"<Requirement 'Kind Requirement' ({TEST_REQ_UUID})>",
+            uuid=TEST_REQ_UUID,
+        ),
+    ]
+    context = [(changes, TEST_MODULE_ID, "1")]
+    tool_metadata = {
+        "revision": "123",
+        "tool": "tool version 1",
+        "connector": "connector v1",
+    }
+    expected_error_msg = (
+        "Can't match class name in short representation: " + TEST_FOLDER_UUID
+    )
+
+    reporter = auditing.RMReporter(migration_model)
+    for changes, module_id, module_category in context:
+        reporter.store_changes(changes, module_id, module_category)
+
+    with pytest.raises(ValueError, match=expected_error_msg):
+        reporter.create_commit_message(tool_metadata)
+
+
+@pytest.mark.integtest
+def test_get_change_report(migration_model: capellambse.MelodyModel):
+    req = migration_model.by_uuid(TEST_REQ_UUID)
+    req.parent.requirements.remove(req)
+    context = [
+        (TestRMReporter.CHANGES, "1", "category1"),
+        (TEST_CHANGES, TEST_MODULE_ID, "category2"),
+    ]
+
+    reporter = auditing.RMReporter(migration_model)
+    for changes, module_id, module_category in context:
+        reporter.store_changes(changes, module_id, module_category)
+    change_report = reporter.get_change_report()
+
+    assert change_report.startswith(
+        "<Folder 'Folder' (e16f5cc1-3299-43d0-b1a0-82d31a137111)>\n"
```

Comment:
IMO it would make more sense (i.e.: look better, and be easier readable) to have this bunch of text as a verbatim text block plus `textwrap.dedent`, similar to:

```
textwrap.dedent(
    """\
    <Folder...
    =====================

    <Folder...
    =====================
    """
```

The extra empty lines (instead of collapsing them into two consecutive `\n` on the end of a single line) do help readability a lot, and make it visually much more similar to the final output (which will show up with actual empty lines). In fact, first time I saw this test, I was briefly confused by the empty lines seemingly missing here, before I saw the `\n` characters at the end.